ingester.yaml

```diff
@@ -0,0 +1,60 @@
+openapi: 3.0.3
+info:
+  title: Ingester
+  description: |-
+  termsOfService: http://swagger.io/terms/
```

Comment:
As an example: change to a different URL

Reply:
What should I change it to?